launcher/ui/pages/global/AccountListPage.cpp

```diff
@@ -94,7 +94,7 @@ AccountListPage::AccountListPage(QWidget *parent)
     updateButtonStates();
 
     // Xbox authentication won't work without a client identifier, so disable the button if it is missing
-    if (APPLICATION->getMSAClientID().isEmpty()) {
+    if (APPLICATION->currentCapabilities() & Application::SupportsMSA) {
```

Comment:
Shouldn't this be the opposite? If it has support for MSA, it shouldn't go into this if, right?